Shruti Bist/Assignment-2/basic-auth/authentication/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+
+# Create your models here.
+class User(models.Model):
```

Comment:
This is not good, User is a pre-defined class, you should name this something else and not User.
Also you don't need to create a User model , it is predefined in django.